siliconcompiler/targets/asap7_demo.py

```diff
@@ -1,4 +1,5 @@
 import siliconcompiler
+from . import utils
```

Comment:
Syntax error. Try `import utils`.

Reply:
there was a separate issue with the docs. The current HEAD of main is green for the build docs. Are they still broken?

Reply:
They're failing on the tip of main if I run `make html` in the `docs/` directory; I think the recent merges were green because the tests ran on the branch pre-merge.